mkdocs/tests/utils/utils_tests.py

```diff
@@ -321,3 +321,30 @@ def test_copy_files_without_permissions(self):
                     os.chmod(src, stat.S_IRUSR | stat.S_IWUSR)
             shutil.rmtree(src_dir)
             shutil.rmtree(dst_dir)
+
+    def test_unicode_clean_directory(self):
+        temp_dir = tempfile.mkdtemp()
+        utf8_directory = os.path.join(temp_dir, '导航')
+        utf8_utf8_directory = os.path.join(utf8_directory, '导航')
+        ascii_directory = os.path.join(temp_dir, 'bar')
+        ascii_ascii_directory = os.path.join(ascii_directory, 'bar')
+
+        os.makedirs(utf8_utf8_directory)
+        os.makedirs(ascii_ascii_directory)
+
+        utf_8_path = os.path.join(utf8_utf8_directory, 'foo.txt')
+        ascii_path = os.path.join(ascii_ascii_directory, 'foo.txt')
+
+        with open(utf_8_path, 'w') as f:
+            f.write('content')
+
+        with open(ascii_path, 'w') as f:
+            f.write('content')
+
+        try:
+            utils.clean_directory(utf8_directory)
+            utils.clean_directory(ascii_directory)
+            self.assertTrue(os.path.exists(utf_8_path), False)
+            self.assertTrue(os.path.exists(ascii_path), False)
```

Comment:
Shouldn't this be:

    self.assertFalse(os.path.exists(utf_8_path))
    self.assertFalse(os.path.exists(ascii_path))

Might also want to add:

    self.assertFalse(os.path.exists(utf8_utf8_directory))
    self.assertFalse(os.path.exists(ascii_ascii_directory))

Reply:
yeah, you're right, I use assertTrue wrong, I use it as assertEqual.